tests/decorators/test_blob.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+import unittest
+
+from azure.functions.decorators.blob import BlobTrigger, BlobOutput, BlobInput
+from azure.functions.decorators.core import BindingDirection, DataType
+
+
+class TestBlob(unittest.TestCase):
+    def test_blob_trigger_valid_creation(self):

```

Comment:
Can you also add a small test where the decorator is actually getting assigned?

Reply:
Essentially - a new blob decorator test in `test_decorators.py`.

Reply:
This would address CodeCov check as well - https://github.com/Azure/azure-functions-python-library/pull/105/checks?check_run_id=5488935060

Reply:
ditto, everything else looks good